README.md

```diff
@@ -112,7 +117,29 @@ while ... { ... }
 ```
 Invariants must have names (for now).
 
-Finally, there is a `variant` expression, which may be useful when defining *logical functions*, whose termination must be proved. You can give it an expression as argument, whose values must strictly decrease (in a known well-founded order) at each recursive call.
+Finally, there is a `variant` expression, which may be useful when defining *logical functions*, whose termination must be proved. You can give it an expression as argument, whose value must strictly decrease (in a known well-founded order) at each recursive call.
```

Comment:
Actually, this is no longer true, but will be once we add `#[pure]` functions. Additionally, I may take a look at enabling termination checking for program functions!